jobbergate-composed/Dockerfile-slurm

```diff
@@ -0,0 +1,156 @@
+FROM centos:7
+
+# Adapted from https://github.com/giovtorres/slurm-docker-cluster
+ARG GOSU_VERSION=1.11
+
+RUN set -ex \
+    && yum makecache fast \
+    && yum -y update \
+    && yum -y install epel-release \
+    && yum -y install \
+       wget \
+       bzip2 \
+       perl \
+       gcc \
+       gcc-c++\
+       git \
+       gnupg \
+       make \
+       munge \
+       munge-devel \
+       python-devel \
+       python-pip \
+       python34 \
+       python34-devel \
+       python34-pip \
+       mariadb-server \
+       mariadb-devel \
+       psmisc \
+       bash-completion \
+       vim-enhanced \
+       autoconf \
+       automake \
+    && yum clean all \
+    && rm -rf /var/cache/yum
+
+RUN ln -s /usr/bin/python3.4 /usr/bin/python3
```

Comment:
Why Python 3.4?

Reply:
Probably just because it's adapted from an old Dockerfile. We could try it with a more modern container, but it works so I'm not inclined to tinker with it much.